Tests/MaterialLib/TestFluidSpecificHeatCapacityModel.cpp

```diff
@@ -0,0 +1,51 @@
+/**
+ *  \copyright
+ *   Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *              Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ *   \file TestFluidSpecificHeatCapacityModel.cpp
+ *
+ */
+
+#include <gtest/gtest.h>
+
+#include <memory>
+
+#include "Tests/TestTools.h"
+
+#include "BaseLib/ConfigTree.h"
+
+#include "MaterialLib/PhysicalConstant.h"
+#include "MaterialLib/Fluid/FluidProperty.h"
+#include "MaterialLib/Fluid/ConstantFluidProperty.h"
+#include "MaterialLib/Fluid/SpecificHeatCapacity/CreateSpecificFluidHeatCapacityModel.h"
+
+using namespace MaterialLib;
+using namespace MaterialLib::Fluid;
+
+using ArrayType = MaterialLib::Fluid::FluidProperty::ArrayType;
+
+std::unique_ptr<FluidProperty> createSpecificFluidHeatCapacityModel(
+    const char xml[])
+{
+    auto const ptree = readXml(xml);
+    BaseLib::ConfigTree conf(ptree, "", BaseLib::ConfigTree::onerror,
+                             BaseLib::ConfigTree::onwarning);
+    auto const& sub_config = conf.getConfigSubtree("specific_heat_capacity");
+    return MaterialLib::Fluid::createSpecificFluidHeatCapacityModel(sub_config);
```

Comment:
Namespaces `MaterialLib::Fluid` in front are not needed since the namespaces are made available in line 26.